package.json

```diff
@@ -19,6 +19,7 @@
     "author": "Visual Studio Code Team",
     "license": "MIT",
     "dependencies": {
+        "hashjs": "^1.2.0",
```

Comment:
FYI this will trigger CG as it's an unapproved crypto lib; I switched to `sha.js`, which is what VS Code uses internally when `crypto` is unavailable.